lib/widgets/conversations/conversation_summaries_page/conversation_summary_tile.dart

```diff
@@ -21,7 +21,7 @@ class ConversationSummaryTile extends StatelessWidget {
         context.dispatch(
             LeaveConversation(conversationId: summary.conversationId));
 
-        Scaffold.of(context).showSnackBar(SnackBar(
```

Comment:
Another attempt to sneak in an out of scope change - the old `Scaffold.of` is deprecated and I wanted to get rid of the analyzer warning 🤞